samples/springmvc-sample/springmvc-consumer/src/main/java/org/apache/servicecomb/samples/springmvc/consumer/SpringmvcConsumerMain.java

```diff
@@ -64,10 +65,25 @@ public static void main(String[] args) throws Exception {
     System.out.println("AsyncRestTemplate Consumer sayHi services: " + responseEntity.getBody());
 
     HttpEntity<Person> entity = new HttpEntity<>(person);
-    ListenableFuture<ResponseEntity<String>> exchange = cseAsyncRestTemplate
+    ListenableFuture<ResponseEntity<String>> listenableFuture = cseAsyncRestTemplate
         .exchange("cse://springmvc/springmvchello/sayhello", HttpMethod.POST, entity, String.class);
-    ResponseEntity<String> responseEntity1 = exchange.get();
-    System.out.println("AsyncRestTemplate Consumer sayHello services: " + responseEntity1.getBody());
+//    ResponseEntity<String> responseEntity1 = listenableFuture.get();
+//    System.out.println("AsyncRestTemplate Consumer sayHello services: " + responseEntity1.getBody());
+
+    listenableFuture.addCallback(
+        new ListenableFutureCallback<ResponseEntity<String>>() {
+          @Override
+          public void onFailure(Throwable ex) {
+            ex.printStackTrace();
```

Comment:
Please printout the exception before printing the stack trace.

Reply:
done!